docs/config.md

```diff
@@ -124,6 +124,16 @@ Which elements can be colored is not yet documented, but see
 the [default color configuration](https://github.com/martinvonz/jj/blob/main/src/config/colors.toml)
 for some examples of what's possible.
 
+### Default command
+
+When `jj` is run with no explicit subcommand, the value of the
+`ui.default-command` setting will used instead. Possible values are any valid
+subcommand name, subcommand alias, or user-defined alias (defaults to `"log"`).
+
+```toml
+ui.default-command = "log"
+```
+
```

Comment:
We should also have a way of disabling the feature, maybe simply by setting the default command to empty string. But that can come later.

Reply:
Setting `ui.default-command = "help"` would essentially restore the previous behavior.

Reply:
Ah, of course. I had thought of that earlier. I think was also thinking that we might need to be able to turn the feature off in case it misbehaved in some cases, but I'm less concerned about that after the rewrite.

Reply:
To nitpick, setting `ui.default-command = "help"` would result in the *long* help being printed, which is different from the short help printed now. I guess that's what you meant by "essentially".

Reply:
Actually, `ui.default-command = "-h"` might work.

Reply:
Yep, sorry...I was thinking that the long help was printed currently, but you're right that `ui.default-command = "-h"` will do what you want.